src/bgen/Generator.cs

```diff
@@ -4392,6 +4395,17 @@ void GenerateMethod (MemberInformation minfo)
 			}
 		}
 
+		if (minfo.is_extension_method) {
+			var property = GetProperty (minfo.Method);
+			if (property is not null) {
+				WriteDocumentation (property);
+			} else {
+				WriteDocumentation (minfo.Method);
+			}
+		} else {
+			WriteDocumentation (minfo.Method);
+		}
```

Comment:
nit but otherwise lgtm!

```suggestion
		if (minfo.is_extension_method) {
			WriteDocumentation (GetProperty (minfo.Method) ?? minfo.Method);
		} else {
			WriteDocumentation (minfo.Method);
		}
```